gui/slick/interfaces/default/home.tmpl

```diff
@@ -364,7 +364,10 @@ $myShowList.sort(lambda x, y: cmp(x.name, y.name))
 					}
 					if (classvalue>80) {
```

Comment:
Could be, but doesn't do much. In my code, the classtoadd variable just gets overridden when it's a 100. Nothing wrong with that.

The old code should indeed be >=. When it's 80, no color is selected. Maybe that's also the cause of issue sickrage-issues/#303